pyqt-apps/siriushla/li_rf_llrf/main.py

```diff
@@ -1,319 +1,194 @@
+            -Power and Phase information
+            -Chart buttons
+        """
+        group = QGroupBox()
+        bi_vlay = QVBoxLayout()
+        bi_vlay.setContentsMargins(2, 2, 2, 2)
+
+        for chart_type in info["Chart"]:
+            bi_vlay.addWidget(
+                self.showChartBtn(
+                    device, channel, chart_type),
+                alignment=Qt.AlignCenter)
+
+        for basic_info in ["Power", "Phase"]:
+            bi_vlay.addLayout(
+                self.baseWidget(
+                    basic_info,
+                    self.buildPvName(
+                        channel, device,
+                        "GET_", '_' + basic_info.upper())))
+        group.setLayout(bi_vlay)
+        group.setStyleSheet('''
+            max-width: 160px; max-height: 150px;
+        ''')
+
+        rel_wid = RelativeWidget(
+            parent=self.image_container,
+            widget=group,
+            relative_pos=info["Position"])
+        self.relative_widgets.append(rel_wid)
+
+    def motorControlBtn(self, device, info):
+        """Show the motor control button"""
+        btn = QPushButton("Motor Control")
+        _util.connect_window(
+            btn, MotorControlWindow,
+            parent=self, motor_type=device)
+        rel_wid = RelativeWidget(
+            parent=self.image_container,
+            widget=btn,
+            relative_pos=info["Position"])
+        self.relative_widgets.append(rel_wid)
+
+    def buildDevicesWidgets(self):
+        """Build and arrange the basic information
+        and control buttons on the image"""
+
+        for device, channel in BASIC_INFO.items():
+            for pv_name, info in channel.items():
+                if device != "MOTOR":
+                    self.basicInfoBox(device, pv_name, info)
+                else:
+                    self.motorControlBtn(pv_name, info)
+
+    def buildLine(self):
+        line = QFrame()
+        line.setFrameShape(QFrame.HLine)
+        line.setFrameShadow(QFrame.Raised)
+        line.setLineWidth(3)
+        return line
+
+    def buildChartMonitor(self):
+        widget = QWidget()
+        lay = QVBoxLayout()
+        for dev in DEVICES:
+            lay.addWidget(
+                ChartMon(
+                    parent=self, dev=dev.pvname, prefix=self.prefix))
+            if dev != DEVICES.SHB:
+                lay.addWidget(self.buildLine())
+        widget.setLayout(lay)
+        return widget
+
+    def buildTabs(self):
+        tab = QTabWidget()
+        tab.setObjectName("LITab")
+        tab.setContentsMargins(0, 0, 0, 0)
+        tab.addTab(self.imageViewer(), "Planta Completa")
+        tab.addTab(self.buildChartMonitor(), "Monitoramento dos Loops de Controle")
+
+        return tab
+
     def _setupui(self):
         """."""
-        wid = QWidget(self)
-        self.setCentralWidget(wid)
         lay1 = QGridLayout()
-        wid.setLayout(lay1)
+        self.setLayout(lay1)
```

Comment:
separate controlbox column in a different layout to add a spacer to help with graph line and control box alignment:
```
    def _setupui(self):
        lay1 = QGridLayout()
        self.setLayout(lay1)

        lay1.addWidget(self.buildTabs(), 0, 1)
        self.buildDevicesWidgets()

        conlay = QGridLayout()
        conlay.addItem(
            QSpacerItem(1, 12, QSizePolicy.Ignored, QSizePolicy.Fixed), 0, 0)
        for dev in DEVICES:
            grbox = QGroupBox(dev.label, self)
            lay = QGridLayout()
            lay.setContentsMargins(0, 0, 0, 0)
            grbox.setLayout(lay)
            lay.addWidget(
                ControlBox(
                    grbox, dev.pvname, main_dev=self.main_dev,
                    device=dev, prefix=self.prefix),
                0, 0)
            conlay.addWidget(grbox, dev.value+1, 0)
        lay1.addLayout(conlay, 0, 0)